portalnet/src/storage.rs

```diff
@@ -444,6 +451,7 @@ impl PortalStorage {
                 debug!("Error writing content ID {id_to_remove:?} to meta db. Reverted: {err:?}",);
             } else {
                 num_removed_items += 1;
+                self.metrics.decrease_entry_count();
```

Comment:
I think this ought to go at the bottom of `evict` since that method is public and the container for deletion logic

Reply:
Ah yes, good catch!